src/hypergraph/graph/validation.py

```
@@ -0,0 +1,247 @@
+        return
+
+    first_value, first_node = with_default[0]
+    for value, node_name in with_default[1:]:
+        if value != first_value:
+            raise GraphConfigError(
+                f"Inconsistent defaults for '{param}'\n\n"
+                f"  -> Node '{first_node}' has default: {first_value!r}\n"
+                f"  -> Node '{node_name}' has default: {value!r}\n\n"
+                f"How to fix:\n"
+                f"  Use the same default in both nodes"
+            )
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)